app/App/Panel/Main/Account/SignerStatus/index.js

```diff
@@ -76,7 +76,9 @@ class SignerStatus extends React.Component {
           this.props.hideSignerStatus(true)
         }
       })
-      this.inputRef.current.focus()
+      if (this.inputRef.current) {
```

Comment:
why is the ref undefined here?

Reply:
Yeah I was being lazy here and simply working around the error. It's because the input doesn't always render, the component render method returns null when there is a hardware signer and it is unlocked.  What should probably happen is the logic for whether this component should render or not, should be in the parent component, which is a more React-y pattern.